src/index.html

```diff
@@ -11,6 +11,37 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <!-- <h1>Search bar airbnb</h1> -->
+    <div class="search-bar-big">
+      <div class="search-container-big">
+        <form id="big" 
+        action="https://localhost:3000/api" 
+        method="post" >
+          <div class="search-image">
+            <label for = "keypress"></label>
+            <input id="keypress" 
+            class="search-field-big" 
+            type="text" name="search" 
+            placeholder="Try &quot;Los Angeles&quot;">
+          </div>
+        </form>
+      </div> 
+    </div>
+    <div class="search-bar-small">
```

Comment:
Add empty lines between multiline sibling blocks of HTML

Reply:
add empty lines after body and mane divs